{"name": "TC1_004: Signup_with_invalid_data via API", "status": "passed", "steps": [{"name": "Step 1: Submit POST request with invalid data.", "status": "passed", "start": 1737005063845, "stop": 1737005063845}, {"name": "Step 2: Verify failed Signup.", "status": "passed", "attachments": [{"name": "API Response", "source": "f864f121-1d78-43e0-a50e-16ef5751b4d2-attachment.json", "type": "application/json"}], "start": 1737005063845, "stop": 1737005064810}], "attachments": [{"name": "logs", "source": "d7a90a66-6800-46b0-8b8f-59072f99cf4d-attachment.txt", "type": "text/plain"}], "start": 1737005063845, "stop": 1737005064810, "uuid": "09ca1c3f-882f-44f6-b3d1-a8b67b85fadb", "historyId": "f1c067eb43dbb2d5ba79be6986a47ccf", "testCaseId": "f1c067eb43dbb2d5ba79be6986a47ccf", "fullName": "tests.api.test_signup_via_api.TestUserSignupFunctionality#test_failed_signup_using_invalid_data", "labels": [{"name": "suite", "value": "User SignUp Functionality"}, {"name": "parentSuite", "value": "Regression"}, {"name": "feature", "value": "User SignUp Functionality"}, {"name": "severity", "value": "normal"}, {"name": "subSuite", "value": "TC1_004: Signup_with_invalid_data via API"}, {"name": "tag", "value": "tags('api_signup_functionality')"}, {"name": "host", "value": "CHEQ-J7M1HB3"}, {"name": "thread", "value": "11780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_signup_via_api"}]}